snowshu/adapters/target_adapters/base_target_adapter.py

```diff
@@ -178,6 +178,17 @@ def initialize_replica(self,
     def _init_image(self,
                     source_adapter_name: str) -> None:
         shdocker = SnowShuDocker()
+
+        image_architecture = shdocker.get_docker_image_attributes(self.DOCKER_IMAGE) \
+            .get('Architecture', LOCAL_ARCHITECTURE)
```

Comment:
What case does defaulting to the `LOCAL_ARCHITECTURE` value support? Defaulting to that value ensures that a missing `"Architecture"` key results in no mismatch between image and host architectures.

Reply:
let's put the 'Unknown' as the default - anyway we treat it as a "warning" message